packages/desktop/components/popups/send/SendFormPopup2.svelte

```diff
@@ -0,0 +1,97 @@
+<script lang="ts">
+    import { get } from 'svelte/store'
+    import { localize } from '@core/i18n'
+    import { selectedAccount } from '@core/account'
+    import {
+        isReservedTagKeyword,
+        InclusionState,
+        selectedAccountActivities,
+        newTransactionDetails,
+        NewTransactionType,
+    } from '@core/wallet'
+    import { getByteLengthOfString } from '@core/utils'
+    import { ownedNfts } from '@core/nfts'
+    import SendTokenForm from './SendTokenForm.svelte'
+    import SendNftForm from './SendNftForm.svelte'
+    import { OptionalInputType, SendFormTab } from '@core/wallet/utils/send/sendUtils'
+    import { FontWeight, Tabs, Text, TextType } from '@ui'
+
+    const tabs: SendFormTab[] = [SendFormTab.SendToken, SendFormTab.SendNft]
+    const { type: transactionType } = get(newTransactionDetails)
+    let activeTab: SendFormTab =
+        transactionType === NewTransactionType.NftTransfer ? SendFormTab.SendNft : SendFormTab.SendToken
```

Comment:
This fixed the issue of Nft tab being open when starting send flow from Collection page and also the Nft in question is selected in the nftInput :+1: 
Issue now is that if you switch to send token tab and than back to nft tab the `nftInput` is reset
![image](https://github.com/iotaledger/firefly/assets/52735957/6439cec0-c0f2-4f7a-8b64-1a59b071500a)